Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Inlines/ImageInline.cs

```diff
@@ -116,14 +130,48 @@ internal static Common.InlineParseResult Parse(string markdown, int start, int e
                 pos++;
             }
 
-            string url = TextRunInline.ResolveEscapeSequences(markdown, urlStart, pos);
+            var imageDimensionsPos = markdown.IndexOf(" =", urlStart, pos - urlStart, StringComparison.Ordinal);
+
+            var url = imageDimensionsPos > 0
+                ? TextRunInline.ResolveEscapeSequences(markdown, urlStart, imageDimensionsPos)
+                : TextRunInline.ResolveEscapeSequences(markdown, urlStart, pos);
+
+            int imageWidth = 0;
+            int imageHeight = 0;
+
+            if (imageDimensionsPos > 0)
+            {
+                // trying to find 'x' which separates image width and height
+                var dimensionsSepatorPos = markdown.IndexOf("x", imageDimensionsPos + 2, pos - imageDimensionsPos - 1, StringComparison.Ordinal);
+
+                // didn't find separator, trying to parse value as imageWidth
+                if (dimensionsSepatorPos == -1)
+                {
+                    var imageWidthStr = markdown.Substring(imageDimensionsPos + 2, pos - imageDimensionsPos - 2);
+
+                    int.TryParse(imageWidthStr, out imageWidth);
+                }
+                else
+                {
+                    var dimensions = markdown.Substring(imageDimensionsPos + 2, pos - imageDimensionsPos - 2).Split('x');
+
+                    // got width and height
+                    if (dimensions.Length == 2)
+                    {
+                        int.TryParse(dimensions[0], out imageWidth);
+                        int.TryParse(dimensions[1], out imageHeight);
+                    }
+                }
+            }
 
             // We found something!
             var result = new ImageInline
             {
                 Tooltip = tooltip,
                 Url = url,
-                Text = markdown.Substring(start, pos + 1 - start)
+                Text = markdown.Substring(start, pos + 1 - start),
+                ImageWidth = imageWidth,
+                ImageHeight = imageHeight
             };
             return new Common.InlineParseResult(result, start, pos + 1);
         }
```

Comment:
ToString should be updated to output the dimensions of the image (if there is one).